test/blackbox-tests/test-cases/install-mld-pages/files-in-documentation-stanza.t/run.t

```diff
@@ -0,0 +1,111 @@
+The files field of the documentation stanza allows to control the hierarchy
+of documentation files
+
+  $ cat doc/dune
```

Comment:
I would avoid outputting the file. It's either in the source tree or if you want to have it in the test then write it in the test, but having it in both places is just duplication. I'd keep it in the source tree.